features/FEATURE_LWIP/lwip-interface/lwipopts.h

```diff
@@ -81,9 +81,9 @@
 #define DEFAULT_ACCEPTMBOX_SIZE     8
 
 #ifdef LWIP_DEBUG
-#define TCPIP_THREAD_STACKSIZE      1024*2
+#define TCPIP_THREAD_STACKSIZE      1200*2
```

Comment:
I'm with @kjbracey-arm, there are other prs to introduce the config system to lwip. Those will need to be updated and can take this option into account:
https://github.com/ARMmbed/mbed-os/pull/2872